libkbfs/folder_block_ops.go

```diff
@@ -2673,12 +2677,24 @@ func (fbo *folderBlockOps) UpdateCachedEntryAttributes(
 	}
 
 	if cleanEntry != nil {
-		fbo.setCachedAttr(ctx, lState, de.ref(), op, cleanEntry)
+		fbo.setCachedAttr(ctx, lState, de.ref(), op, cleanEntry, false)
 	}
 
 	return childNode, nil
 }
 
+// UpdateCachedEntryAttributesOnRemovedFile updates any cached entry
+// for the given path of an unlinked file, according to the given op,
+// and it makes a new dirty cache entry if one doesn't exist yet.  We
+// assume Sync will be called eventually on the corresponding open
+// file handle, which will clear out the entry.
+func (fbo *folderBlockOps) UpdateCachedEntryAttributesOnRemovedFile(
+	ctx context.Context, lState *lockState, md *RootMetadata,
+	file path, op *setAttrOp, de DirEntry) error {
+	fbo.setCachedAttr(ctx, lState, de.ref(), op, &de, true)
```

Comment:
any reason to pass in `md` and `file`?

Also may as well omit the return value.